runtime/doc/deprecated.txt

```diff
@@ -26,7 +26,7 @@ DIAGNOSTICS
 
 HIGHLIGHTS
 
-• todo
+• *:ownsyntax* *w:current_syntax*	Use 'winhighlight' instead.
```

Comment:
?

Reply:
How does 'winhighlight' achieve different syntax rules in different windows?

Reply:
I didn't claim that, but that isn't really necessary. 'winhighlight' can map any syntax rule to a different highlight.

Other than "conceal" (which can already be controlled by window-local 'conceallevel'), when would window-local syntax be able to achive something that window-local highlight can't?